trusted/session_data.html

```diff
@@ -31,9 +41,12 @@
     </div>
     <form class="form-inline" role="form" id="inputform">
       <input type="text" id="sess" placeholder="Session Number"/>
-      <button id="submit" type="button" class="btn btn-primary" onclick="genT()" >SUBMIT</button>
+      <input type="password" id="pass" placeholder="Session Password"/>
+      <button id="submit" type="button" class="btn btn-primary" onclick="genT()">SUBMIT</button>
```

Comment:
I know this code was there before, but we should be more consistent about our coding style. Onclick in html, document.getElementById jQuery, ...